src/facet.ts

```diff
@@ -1,4 +1,7 @@
-import {FacetFieldDef} from './fielddef';
+import {FieldDef} from './fielddef';
+
+// TODO: add more facet properties
+export type FacetFieldDef = FieldDef;
```

Comment:
I fear that it won't be incompatible with other `FieldDef`s for the rest of the codebase.